### PR TITLE
Delete rpcs

### PR DESCRIPTION
**Pull request by Joshdike**

### CODE REVIEW

File: internal/db/pg/pets.go [hunk 1]
```diff
@@ -82,3 +82,19 @@ func (r *Repository) UpdatePet(ctx context.Context, p *gostreamv1.Pet) error {
...
+		PlaceholderFormat(sq.Dollar).ToSql()
+	if err != nil {
+		return errs.B().Code(errs.Internal).Msg("sq error").Err()
+	}
+	if _, err = r.pool.Exec(ctx, query, params...); err != nil {
+		if errors.Is(err, pgx.ErrNoRows) {
+			return errs.B().Code(errs.InvalidArgument).Msg("pet does not exist").Err()
+		}
+		return err
+	}
```
Comment (lordvidex) [inline internal/db/pg/pets.go:98]: like in DeleteUser, wrap this with internal error code.

```
return errs.WrapCode(err, errs.Internal, "database error occurred")
```

**Comment by lordvidex** [comment]
Closes #3

File: internal/db/pg/pets.go (hunk 1 above)
Comment (lordvidex) [reply internal/db/pg/pets.go:98]: Everything else looks good

Comment (Joshdike) [reply internal/db/pg/pets.go]: fixed

**Pull request merged**